lib/ziti-tunnel-cbs/ziti_hosting.c

```diff
@@ -35,10 +35,19 @@
 #endif
 #endif
 
-#define ZITI_MTU (15 * 1024)
-#define MAX_OUTSTANDING_WRITES 8
-
 /********** hosting **********/
+static void on_bridge_close(uv_handle_t *handle);
+
+struct hosted_io_ctx_s {
+    struct hosted_service_ctx_s *service;
+    ziti_connection client;
+    char server_dial_str[64];
+    int server_proto_id;
```

Comment:
Do we actually need `server_proto_id` any more?

Reply:
cleaned up